src/mvTables.cpp

```diff
@@ -72,7 +72,9 @@ void mvTableColumn::handleSpecificKeywordArgs(PyObject* dict)
 	flagop("prefer_sort_descending", ImGuiTableColumnFlags_PreferSortDescending, _flags);
 	flagop("indent_enable", ImGuiTableColumnFlags_IndentEnable, _flags);
 	flagop("indent_disable", ImGuiTableColumnFlags_IndentDisable, _flags);
-
+	flagop("angle_header", ImGuiTableColumnFlags_AngledHeader, _flags);
```

Comment:
:exclamation::exclamation: It doesn't really work, since the table doesn't call `ImGui::TableAngledHeadersRow()`.

Also, I'd name it `angled_header` (angled, not angle) to follow the ImGui name.